gitclient/datadog_checks/gitclient/gitclient.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+from datadog_checks.utils.subprocess_output import get_subprocess_output
+
+
+class GitclientCheck(AgentCheck):
+    def check(self, instance):
```

Comment:
Please use `self.instance` instead of the passed in `instance` a its a deprecated parameter